src/http/request_handler.rs

```diff
@@ -187,6 +226,19 @@ async fn handle_rest_apis(
             app_ctx.as_ref(),
         );
         update_experimental_headers(&mut resp, app_ctx.as_ref(), req_ctx);
+
+        if let Some(body) = get_value_from_response(&mut resp).await {
```

Comment:
We should perform this operation before converting the value to response. GraphQLResponse contains deserialized version of the ConstValue, on which we can quickly pattern match and extract out the things we don't need.